tests/functional/test_screen.py

```diff
@@ -2,16 +2,21 @@
 import sys
 import os.path
 import tempfile
+from shutil import which
 
 import pexpect
+import pytest
 
 
 DATADIR = os.path.join(os.path.dirname(__file__), 'data')
 SIMPLE_PNG  = os.path.join(DATADIR, 'simple.png')
 EXAMPLE_PNG = os.path.join(DATADIR, 'example.png')
 EXAMPLE_NOCURSOR_PNG = os.path.join(DATADIR, 'example_nocursor.png')
 
+SERVER = "exmaple"
```

Comment:
typo?

Reply:
Yes, will fix.
While at it maybe you can clarify how that  _functional_ unit test worked: It looks like you have an executable named `example` in your test  environment as this is `pexcept.spawn()`ed. Maybe it would be more useful to make that executable configurable, e.g. get it from some environment variable via `os.environ.get("VNCDOTOOL_TEST_SERVER", "example")`

Reply:
+1, will add some notes on testing and have a go at making them work in CI too.